### ISSUE TITLE
Killing nc process kills parent process if it contains the letters "nc"

### ISSUE THREAD

**Issue by tgxworld** [Neilpang/acme.sh#164]
I'm executing `acme.sh` inside another script named `letsencrypt`.

However, the parent process is [picked up and killed](https://github.com/Neilpang/acme.sh/blob/998c92d57ee23e441f4bd71c735fa2ba7ce267b5/acme.sh#L750).

```
301 ? 00:00:00 letsencrypt
[Thu Apr 21 02:35:31 UTC 2016] kill 301
Killed
```

**Comment by Neilpang** [comment]
fixed: 0103f59f826ae19972dfbd569f13d21322745c6a
It's unnecessary kill.  the nc server would exit automatically after verification.  Otherwise, the following `_get` call would make it exit on error.

**Comment by tgxworld** [comment]
Awesome! Thank you @Neilpang
